plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotConfig.java

```diff
@@ -58,6 +60,9 @@
     private int maxRowsForBrokerQueries = 50_000;
     private boolean aggregationPushdownEnabled = true;
     private boolean countDistinctPushdownEnabled = true;
+    private boolean grpcEnabled;
```

Comment:
Should this be enabled by default?

Reply:
Yep, less impact on the pinot servers and allows for larger result sets.

Reply:
Pinot servers will have to enable the endpoint in their configuration though, by default it's not enabled. Can add a note in the documentation.

Reply:
I've turned on the grpc flag on pinot by default recently. I think we can make this enabled after one or two versions.